src/sidebar/services/annotations-exporter.tsx

```diff
@@ -152,6 +152,89 @@ export class AnnotationsExporter {
     return `${headers}\n${annotationsContent}`;
   }
 
+  buildHTMLExportContent(
+    annotations: APIAnnotationData[],
+    {
+      groupName = '',
+      displayNamesEnabled = false,
+      defaultAuthority = '',
+      /* istanbul ignore next - test seam */
+      now = new Date(),
+    }: ExportOptions = {},
+  ): string {
+    const { uri, title, uniqueUsers, replies, extractUsername } =
+      this._exportCommon(annotations, {
+        displayNamesEnabled,
+        defaultAuthority,
+      });
+
+    return renderToString(
+      <html lang="en">
+        <head>
+          <title>Annotations export - Hypothesis</title>
+          <meta charSet="UTF-8" />
+        </head>
+        <body>
+          <section>
+            <h1>Summary</h1>
+            <p>
+              <time dateTime={now.toISOString()}>{formatDateTime(now)}</time>
+            </p>
+            <p>
+              <strong>{title}</strong>
+            </p>
+            <p>
+              <a href={uri} target="_blank" rel="noopener noreferrer">
+                {uri}
+              </a>
+            </p>
+
+            <dl>
+              <dt>Group</dt>
+              <dd>{groupName}</dd>
+              <dt>Total users</dt>
+              <dd>{uniqueUsers.length}</dd>
+              <dt>Users</dt>
+              <dd>{uniqueUsers.join(', ')}</dd>
+              <dt>Total annotations</dt>
+              <dd>{annotations.length}</dd>
+              <dt>Total replies</dt>
+              <dd>{replies.length}</dd>
+            </dl>
```

Comment:
`<dl>` is semantically appropriate, but seems the import into Word / Google Docs is not ideal. We might be better off switching to a `<table>` for this.

Reply:
I think a table should work nicely here.